src/sentry/api/endpoints/organization_release_previous_commits.py

```diff
@@ -44,6 +46,14 @@ def get(self, request, organization, version):
             .extra(order_by=["-date"])[:1]
         )
 
+        analytics.record(
+            "release.get_previous_commits",
+            user_id=request.user.id if request.user and request.user.id else None,
+            organization_id=organization.id,
+            project_ids=[project.id for project in release.projects.all()],
+            user_agent=request.META.get("HTTP_USER_AGENT", ""),
```

Comment:
I see we're calling it `user_agent` here but `source` in the definition. Is that intentional?

Reply:
nope, it's an oversight. It should all be `user_agent`. Good catch!